### DISCUSSION TITLE
Are there any plans to support windows older than 10?

### DISCUSSION THREAD

**Discussion by BreyerW** [dotnet/maui#701] [Q&A]
From what i see first version for windows is built with winui 3 which requires win 10 and specific build version at that. Im evaluating between avalonia and maui but if maui is going to be stuck with win 10 only going forward then decision became much simpler (in favour of avalonia of course since avalonia officially supports at least win 8 and above)

**Comment by vhugogarcia** [comment]
I really hope MAUI does not support Windows 8. That is like moving backwards. 

In fact, I would like MUAI to support from iOS 12+, sadly it still will support iOS 10. We are around the corner on iOS 15.

**Reply by BreyerW** [reply]
@vhugogarcia  it may seem going backward but win 7 & 8 still holds ~20-25% WORLDWIDE share among windows (of that it is mostly win 7 since win 8 gave away free upgrade to 10 unlike win 7, also share of win 7 can be cosiderably larger for some countries and regions like asia where share of win 7 is like 30-40%). Concluding from business and practical standpoint support for at least win 7 is still desirable (anything older is in <1% territory and thus ignorable). 

Source i used for numbers [https://gs.statcounter.com/windows-version-market-share/desktop/asia](https://gs.statcounter.com/windows-version-market-share/desktop/asia)

**Reply by vhugogarcia** [reply]
That is not true. You shared the stats from Asia, not the worldwide on that page. 

If you really go to the worldwide usage: https://gs.statcounter.com/windows-version-market-share/desktop/worldwide 
You will find that Windows 8 & 8.2 is around 4% only. Windows 7 is around 17% but getting down every month. Technology must follow tends based on usage, it needs to continue evolving otherwise we will be stuck in the past. 

Besides that on Windows 7, MS ended support back in January 2020. Windows 8 basic support ended and the extended one is just around the corner to end. 

Security and innovation is something I think MAUI has envision.

**Reply by BreyerW** [reply]
Ah sorry intended to link worldwide but forgot to go back when copying. And ~20-25% figure was for both win 7 & 8 (for worldwide it is closer to 21% but even so it is still 1 computer in 5 worldwide which is hard to ignore and for some specific countries thats 1 in 3 computer)

**Reply by AraHaan** [reply]
There are even some other counties where it happens to be that windows 7 is about 66% or more. It just depends on how poor they are. I could argue that there are still users using builds of windows under Windows XP but that is probably about 0.5% worldwide with ATM's being the only computers worldwide running Windows XP last time I looked (it was 2 years ago when I seen them actually service the thing by rebooting it). 

As to the reason they still use Windows XP, I am not sure entirely unless they have code specifically designed for XP and that it won't run on newer versions of windows and is sensitive to "compatibility" mode and then refuses to run if it detects it's being run in "compatibility" mode.

In 2 years time however that could have possibly changed though very easily.

**Comment by juansturla** [comment]
I guess that you won't have MAUI for Windows older than Windows 10, but you can still build Windows Forms project for Windows 7+
![image](https://user-images.githubusercontent.com/10248860/130677845-9fa3cdf5-44cc-456c-8c7e-84d458c50221.png)
